cpp/test/generated/binary/protocols.cc

```diff
@@ -706,7 +706,7 @@ template<typename T1, yardl::binary::Reader<T1> ReadT1, typename T2, yardl::bina
 namespace basic_types::binary {
 namespace {
 template<typename K, yardl::binary::Writer<K> WriteK, typename V, yardl::binary::Writer<V> WriteV>
-[[maybe_unused]] void WriteAliasedMap(yardl::binary::CodedOutputStream& stream, basic_types::AliasedMap<K, V> const& value) {
+[[maybe_unused]] static void WriteAliasedMap(yardl::binary::CodedOutputStream& stream, basic_types::AliasedMap<K, V> const& value) {
```

Comment:
Is the addition of `static` to make the internal linkage explicit (even though the function is in an unnamed namespace) or is there something else?

Reply:
My misconception about compiler optimizations for static vs unnamed namespace. This change has been reverted.